test/snapshots/seam-blueprint.test.ts.md

```diff
@@ -279,6 +321,48 @@ Generated by [AVA](https://avajs.dev).
           path: '/acs/systems',
           subroutes: [],
         },
+        {
+          endpoints: [
+            {
+              codeSamples: [],
+              deprecationMessage: '',
+              description: '',
+              isDeprecated: false,
+              isUndocumented: false,
+              parameters: [],
+              path: '/acs/systems/list_compatible_credential_manager_acs_systems',
+              request: {
+                methods: [
+                  'POST',
+                ],
+                parameters: [
+                  {
+                    deprecationMessage: '',
+                    description: '',
+                    format: 'uuid',
+                    isDeprecated: false,
+                    isRequired: true,
+                    isUndocumented: false,
+                    name: 'acs_system_id',
+                    type: 'string',
+                  },
+                ],
+                preferredMethod: 'POST',
+                semanticMethod: 'POST',
+              },
+              response: {
+                description: 'OK',
+                responseType: 'void',
+              },
+              title: '',
+            },
+          ],
+          namespace: {
+            path: '/acs',
+          },
+          path: '/acs/systems',
+          subroutes: [],
+        },
```

Comment:
Is it supposed to be like this:
```
{
  routes: [
    {
      path: '/acs/systems',
      endpoints: [
        {
          path: '/acs/systems/get',
        },
      ],
    },
    {
      path: '/acs/systems',
      endpoints: [
        {
          path: '/acs/systems/list',
        },
      ],
    },
    {
      path: '/acs/systems',
      endpoints: [
        {
          path: '/acs/systems/list_compatible_credential_manager_acs_systems',
        },
      ],
    },
  ],
}
```
Wouldn't it make more sense to do like this instead:
```
{
  routes: [
    {
      path: '/acs/systems',
      endpoints: [
        {
          path: '/acs/systems/get',
        },
        {
          path: '/acs/systems/list',
        },
        {
          path: '/acs/systems/list_compatible_credential_manager_acs_systems',
        },
      ],
    },
  ],
}
```

Reply:
Created a PR fixing this: https://github.com/seamapi/blueprint/pull/56